src/fosslight_reuse/_result.py

```diff
@@ -225,13 +225,13 @@ def result_for_summary(oss_pkg_info_files, license_missing_files, copyright_miss
     license_missing_files = list(set(license_missing_files) - set(lic_present_files_in_yaml))
```

Comment:
How about merging with line# 225 and 229 ?
ex)     license_missing_files = list(set(license_missing_files) - set(lic_present_files_in_yaml) - set(excluded_files))